### PR TITLE
Fix indents for 2nd example

### PR DESCRIPTION
**Pull request by zasca**
##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by nitzmahone** [comment]
Both indentation styles are valid, and this change makes the file internally inconsistent with the other indentation above. The addition of `tasks` is good, but we'll probably pass on the indentation change.

**Review by samccann — DISMISSED** [review]

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 1]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
-        - name: Restart memcached
-          ansible.builtin.service:
-            name: memcached
-            state: restarted
-
-        - name: Restart apache
-          ansible.builtin.service:
-            name: apache
-            state: restarted
+    handlers:
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
      handlers:
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 2]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+    - name: Restart memcached
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
        - name: Restart memcached
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 3]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+      ansible.builtin.service:
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
          ansible.builtin.service:
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 4]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+        name: memcached
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
            name: memcached
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 5]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+        state: restarted
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
            state: restarted
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 6]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+
+    - name: Restart apache
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
        - name: Restart apache
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 7]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+      ansible.builtin.service:
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
          ansible.builtin.service:
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 8]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+        name: apache
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
            name: apache
```

File: docs/docsite/rst/user_guide/playbooks_handlers.rst [hunk 9]
```diff
@@ -54,16 +55,16 @@ In this example playbook, the second task notifies the handler. A single task ca
...
+        state: restarted
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_handlers.rst]:
```suggestion
            state: restarted
```

**Review by samccann — COMMENTED** [review]
attempting to remove the uneccessary indenting.

**Review by samccann — APPROVED** [review]

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @zasca for the Ansible docs fix!  We talk about Ansible documentation on maxtrix at [#docs:ansible.im](https://matrix.to/#/#docs:ansible.im) and on libera IRC at #ansible-docs if you ever want to join us and chat about the docs!

**Pull request merged**